app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -11,14 +11,19 @@ description: |
 <%
   paths = [
     '/EFI/fedora/shim.efi',
+    '/EFI/fedora/shimx64.efi',
```

Comment:
:thinking: Maybe only the `x64.efi` paths are necessary, we might save some time by having only one variant

Reply:
Well, it will definitely reduce the file size. :+1: from me to that.
Let's not be crazy and just do a basic per-distribution if-else block.